src/main/java/emu/grasscutter/command/commands/QuestCommand.java

```diff
@@ -11,14 +11,17 @@
 @Command(
         label = "quest",
         aliases = {"q"},
-        usage = {"(add|finish) [<questId>]"},
+        usage = {
+            "(add|finish|running|talking|debug|triggers|grouptriggers) [<questId>]",
+            "dungeons"        
+        },
         permission = "player.quest",
         permissionTargeted = "player.quest.others")
 public final class QuestCommand implements CommandHandler {
 
     @Override
     public void execute(Player sender, Player targetPlayer, List<String> args) {
-        if (args.size() != 2) {
+        if (args.size() != 2  || (args.size() == 1 && args.get(0).toLowerCase() != "dungeons")) {
```

Comment:
String's `!=` compares the object reference, not the content. Use `equals()` instead.

```suggestion
        if (args.size() != 2  || (args.size() == 1 && !args.get(0).toLowerCase().equals("dungeons")))
```

Reply:
BTW, looks like command `/quest dungeons` never works. It's going to hit the exception below (line 33) before entering the switch block:

```java
        try {
            questId = Integer.parseInt(args.get(1));
        } catch (Exception e) {
            CommandHandler.sendMessage(sender, translate(sender, "commands.quest.invalid_id"));
            return;
        }
```